sphinx/source/docs/user_guide/tools.rst

```diff
@@ -42,7 +42,10 @@ Valid values are:
 If you would like to hide the toolbar entirely, pass ``None``.
 
 Below is some code that positions the toolbar below the plot. Try
-running the code and changing the ``toolbar_location`` value.
+running the code and changing the ``toolbar_location`` value. Note
+that the toolbar position clashes with the default axes, in this case
+setting the ``toolbar_sticky`` option to ``False`` will move the toolbar
+to outside of the region where the axis is drawn.
```

Comment:
If you are up for it, what might be ideal is to change the script referenced below to have two plots in a `row`, identical except with one that has `toolbar_sticky=False`